src/ResourceManager/Version2016_09_01/Utilities/SubscriptionExtensions.cs

```diff
@@ -29,6 +30,7 @@ public static AzureSubscription ToAzureSubscription(this Subscription other, IAz
             subscription.SetEnvironment(environment != null ? environment.Name : EnvironmentName.AzureCloud);
             subscription.SetHomeTenant(other.TenantId ?? retrievedByTenant);
             subscription.SetTenant(retrievedByTenant);
+            subscription.SetSubscriptionPolicies(JsonConvert.SerializeObject(other.SubscriptionPolicies));
```

Comment:
Why are you setting the serialized object instead the raw object into subscription?

Reply:
Because I stored subcriptionPolices in extendedProperty, whose type is IDictionary<string,string>

Reply:
Makes sense